addons/point_of_sale/static/src/js/ChromeWidgets/ClientScreenButton.js

```diff
@@ -38,7 +38,7 @@ odoo.define('point_of_sale.ClientScreenButton', function(require) {
         async onClickProxy() {
             try {
                 const renderedHtml = await this.env.pos.render_html_for_customer_facing_display();
-                const ownership = await this.env.pos.proxy.take_ownership_over_client_screen(
+                let ownership = await this.env.pos.proxy.take_ownership_over_client_screen(
                     renderedHtml
                 );
                 if (typeof ownership === 'string') {
```

Comment:
@caburj You might want to take a look at this, this variable is const but is reassigned in the next if. Either that if is dead code, or it's broken and not tested.